packages/utilities/components/min.ts

```diff
@@ -0,0 +1,16 @@
+const compare = (a, b) => {
+  return b < a ? b : a;
+};
+function min(a?: number | Date | string, b?: number | Date | string) {
```

Comment:
Looks like we can compare string with Date, or Number with a string. Is this intended?

Reply:
Yes, well we are able to compare them however in a reasonable manner it only makes sense to compare Date's and numbers if you think that a number could be a timestamp. With the strings, it only makes sense to compare them with strings.